integration/benchmark/cmd/ci/main.go

```diff
@@ -281,3 +282,18 @@ func sendDataToBigQuery(
 	}
 	return nil
 }
+
+func outputLocalData(slices []dataSlice) {
+	jsonText, err := json.MarshalIndent(slices, "", "    ")
+	if err != nil {
+		println("Error converting slice data to json")
+	}
+
+	// output human-readable json blob
+	timestamp := time.Now()
+	fileName := fmt.Sprintf("tps-results-%v.json", timestamp.Format("2006-02-01 15:04"))
+	err = os.WriteFile(fileName, jsonText, 0666)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
this is going to silently ignore the json marshaling error - can we use log.Fatal() instead?

Reply:
Done